Mailer/TwigSwiftMailer.php

```diff
@@ -40,9 +41,9 @@ class TwigSwiftMailer implements MailerInterface
     protected $parameters;
 
     /**
-     * TwigSwiftMailer constructor.
+     * @param \Twig_Environment|Environment $twig
```

Comment:
Use the namespace name all the time instead. It is available in 1.x, 2.x and 3.x.

Reply:
Sorry. I did not notice that in the dependencies Twig 1.28